tests/profiles/cff_test.py

```diff
@@ -62,3 +62,20 @@ def test_check_cff2_call_depth():
     assert_results_contain(check(font),
                            FAIL, 'recursion-error',
                            'Recursion error while decompiling glyph "F".')
+
+
+def test_check_cff_deprecated_operators():
+    check = TestingContext(cff_profile,
+                           "com.adobe.fonts/check/cff_deprecated_operators")
+
+    # this font uses the deprecated 'dotsection' operator
+    font = TEST_FILE('deprecated_operators/cff1_dotsection.otf')
+    assert_results_contain(check(font),
+                           WARN, 'deprecated-operator-dotsection',
+                           'Glyph "i" uses deprecated "dotsection" operator.')
+
+    # this font uses the deprecated 'seac' operator (special case of 'endchar')
+    font = TEST_FILE('deprecated_operators/cff1_seac.otf')
+    assert_results_contain(check(font),
+                           FAIL, 'deprecated-operator-seac',
+                           'Glyph "Aacute" uses deprecated "seac" (endchar) operator.')
```

Comment:
after you de-duplicate the message keyword, please make sure you add here the other needed test-case

Reply:
See response above. I have removed the code and check for explicit `seac` and renamed the test file to match the actual condition (deprecated _use_ of `endchar`, behaving as `seac`) in https://github.com/googlefonts/fontbakery/pull/3033/commits/66b8ecf818c9ec236bdb0fdf75a3bcde982f8b51